uber/custom_tags.py

```diff
@@ -518,6 +518,21 @@ def render(self, context):
         return '<div class="prereg-price-notice">{}</div>'.format(notice) if notice else ''
 
 
+@tag
+class table_prices(template.Node):
+    def render(self, context):
+        if len(c.TABLE_PRICES) <= 1:
+            return '${} per table'.format(c.TABLE_PRICES['default_price'])
+        else:
+            cost, costs = 0, []
+            for i in range(1, 1 + c.MAX_TABLES):
```

Comment:
``c.MAX_TABLES`` is 4, and the Python ``range`` operator is exclusive of the second argument, so ``range(1, 4)`` would return an iterator with ``[1, 2, 3]`` but ``range(1, 5)`` will return an iterator with ``[1, 2, 3, 4]`` which is what we want.